screen_objects.json

```diff
@@ -0,0 +1,80 @@
+{
```

Comment:
@VladimirMakaev I disagree with this strongly. I don't think there's any reason to keep Locator or ScreenObject.

With this new approach, if we wanted to add a new screen object then we can add it to our json config and then it will "just work" (i.e., you'll be able to say ui_manager.SCREEN_OBJECTS['MyNewScreenObject'].

With what you guys have, you'd have to create new classes and push code; I don't understand the value add to the existing architecture, but I'm open to discussion.

Reply:
This is exactly the opposite from the idea. If that's becomes a dictionary of locators then it's just better to hard code it as such. There is absolutely zero value in having json. 
e.g
```python
class Locators:
     PlayBtn = Locator(...)
     MainMenu = Locator(...)
```
At least you'd get IDE helping you against syntax erros

Original idea was to have objects that encapsulate parts of the ui and have behavior & state attached to those parts of the UI. If we see there is no behavior then just have those in Python. I'm not really sure what you're achieving with your statement of "just work" since you still have to write functions that make sense of those locators and declaring a locator in code or in json it's essentially the same with a downside of not being a part of the language